usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -139,7 +139,10 @@ def handle(self, *args, **options):
         # Once all the files have been processed, run any global
         # cleanup/post-load tasks.
         # 1. Load subawards
-        load_subawards(submission_attributes, db_cursor)
+        try:
```

Comment:
Yeah, it feels strange to treat the subaward load process as "it's okay if this fails" when every other part of a submission load is critical enough that we can't consider the submission done if it fails. That said, I think this is a question for the larger group...will ping the channel.